python/pyarrow/_dataset_parquet.pyx

```diff
@@ -613,6 +613,9 @@ cdef class ParquetFileWriteOptions(FileWriteOptions):
             write_page_index=self._properties["write_page_index"],
             write_page_checksum=self._properties["write_page_checksum"],
             sorting_columns=self._properties["sorting_columns"],
+            store_decimal_as_integer=(
```

Comment:
why a `()` is added?

Reply:
I believe that was a linter/formatter recommendation, but its certainly not necessary

Reply:
The core and `test_basic.py` LGTM, but I'm not sure this part ( _dataset ) is tested, I'll check that tomorrow

Reply:
> why a `()` is added?

My latest commit removed this